.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - "charts/**"
```

Comment:
A new chart will be released each time we push changes to `charts`. Right?

Reply:
yes, same behavior, just blocking when there is no changes in the chart, to we be able to update other things like ci and configs